--- autoconf-2.69/build-aux/config.sub	2012-04-25 14:37:26.000000000 +1200
+++ config.sub	2019-12-26 12:51:37.663118210 +1300
@@ -1368,7 +1368,7 @@
 	      | -os2* | -vos* | -palmos* | -uclinux* | -nucleus* \
 	      | -morphos* | -superux* | -rtmk* | -rtmk-nova* | -windiss* \
 	      | -powermax* | -dnix* | -nx6 | -nx7 | -sei* | -dragonfly* \
-	      | -skyos* | -haiku* | -rdos* | -toppers* | -drops* | -es*)
+	      | -skyos* | -haiku* | -rdos* | -toppers* | -drops* | -es* | -qword* )
 	# Remember, each alternative MUST END IN *, to match a version number.
 		;;
 	-qnx*)
